### DISCUSSION TITLE
Usability / Feature request - assign aliases to wallet keys

### DISCUSSION THREAD

**Discussion by forReason** [Chia-Network/chia-blockchain#11124] [Ideas]
When starting the chia application one is prompted to choose the key / wallet which they want to open.

Having more then one wallet, things start to get complicated.
![image](https://user-images.githubusercontent.com/12736950/162688139-d17b47c4-5294-4bf8-b829-276eba0c1f5e.png)

The wallet only have cryptic numbers as names which are quite simmilar, I always have to look up, which number is according to my mining wallet, which my "daily use" wallet and which is my savings wallet. Things get even more complicated if I have a wallet for my business and one personal wallet.

It would be really helpful to be able to assign wallet names or aliases to recognize the wallets without looking the number up.

**Comment by centrd** [comment]
Dup of #10817
